src/main/java/seedu/duke/logic/commands/circuit/TutorialCircuitCommand.java

```diff
@@ -1,21 +1,25 @@
 package seedu.duke.logic.commands.circuit;
 
 import seedu.duke.DukeException;
+import seedu.duke.logic.commands.SummaryCommand;
 import seedu.duke.logic.commands.TutorialCommand;
 import seedu.duke.logic.parser.Parser;
 import seedu.duke.ui.Ui;
 
 import java.util.logging.Level;
 
-public class TutorialCircuitCommand extends CircuitCommand implements TutorialCommand {
+public class TutorialCircuitCommand extends SummaryCommand implements TutorialCommand {
+
     private static final Parser PARSER = new Parser();
     private static final String[] orderOfInstructions = {"template", "set v", "set", "set", "add", "calc"};
     private int numOfCommandsDone = 0;
     private final Ui ui;
+    protected SummaryCommand summary;
```

Comment:
I don't think we need to instantiate anymore since we are inheriting SummaryCommand here

Reply:
👍 
```suggestion
```